spec/models/concerns/journaled/changes_spec.rb

```diff
@@ -43,14 +44,16 @@ def trigger_after_destroy_hooks
 
   subject { klass.new }
 
-  let(:change_writer) { double(Journaled::ChangeWriter, create: true, update: true, delete: true) }
+  let(:change_writer) { instance_double(Journaled::ChangeWriter, create: true, update: true, delete: true) }
 
   before do
+    # rubocop:disable RSpec/ExpectInHook
     allow(Journaled::ChangeWriter).to receive(:new) do |opts|
       expect(opts[:model]).to eq(subject)
       expect(opts[:change_definition].logical_operation).to eq(:change_of_heart)
       change_writer
     end
+    # rubocop:enable RSpec/ExpectInHook
```

Comment:
Decided to just throw some ignores here. While the `expect` is written in the `before` it actually gets executed with you call the `new` method and as such I thought it was ok to just ignore the cop

Reply:
Yeah, good enough for gov't work. :) This is my bad for sure, but it doesn't seem like a great sin in context.